pkg/cmd/config/config.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2024 The KitOps Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+package config
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"kitops/pkg/output"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
+)
+
+type Config struct {
+	LogLevel  string `json:"log_level"`
+	Progress  string `json:"progress"`
+	ConfigDir string `json:"config_dir"`
```

Comment:
Is it possible for the configDir setting to have any effect? The config file is read from the config dir, so setting a different config dir in the file seems strange.

Reply:
Yeah you are correct! This can be removed to avoid confusion in the future